pygmt/datasets/earth_relief.py

```diff
@@ -10,8 +10,8 @@
 from pygmt.src import grdcut, which
 
 
-@kwargs_to_strings(region="sequence")
-def load_earth_relief(resolution="01d", region=None, registration=None):
+@kwargs_to_strings(convert_bools=False, region="sequence")
+def load_earth_relief(resolution="01d", use_srtm=False, region=None, registration=None):
```

Comment:
Move `use_strm` to the end, because it's less used than other parameters.
```suggestion
def load_earth_relief(resolution="01d", region=None, registration=None, use_srtm=False):
```